charts/self-host/templates/admin.yaml

```diff
@@ -98,6 +102,11 @@ spec:
           persistentVolumeClaim:
             claimName: {{ default ( include "bitwarden.applogs" . ) .Values.volume.logs.existingClaim }}
         {{- end }}
+        {{- if .Values.volume.ca_certificates.enabled }}
+        - name: ca-certificates
+          persistentVolumeClaim:
```

Comment:
I think a ConfigMap or Secret mount would suit (at least our use case with the certificat mounting) better.